### PR TITLE
add ownership changes to the editor tools (bug 712682)

### PR DESCRIPTION
**Pull request by magopian**
Fixes [bug 712682](https://bugzilla.mozilla.org/show_bug.cgi?id=712682)

Misses tests:
- for the display of the user changes to the editor tools
- for the sending of emails to authors of the addon

screen shot 2014-12-24 at 16 43 53

### CODE REVIEW

**Comment by magopian** [comment]
@kmaglione r?

File: apps/devhub/views.py [hunk 1]
```diff
@@ -405,16 +406,39 @@ def ownership(request, addon_id, addon):
...
+        authors_emails = set(addon_authors_emails +
+                             [author.user.email for author in authors])
         for author in authors:
             action = None
             if not author.id or author.user_id != author._original_user_id:
                 action = amo.LOG.ADD_USER_WITH_ROLE
                 author.addon = addon
+                mail_user_changes(
+                    author=author,
+                    title=_('An author has been added to your addon'),
```
Comment (clouserw) [inline apps/devhub/views.py]: "addon" should be "add-on" everywhere that it is shown

File: apps/editors/templates/editors/review.html [hunk 2]
```diff
@@ -35,6 +35,18 @@ <h4 class="author">{{ _('by') }} {{ users_list(addon.listed_authors) }}</h4>
 
 {% include 'addons/details_box.html' %}
 
+{% if user_changes %}
+<div id="user-changes-history">
+  <h3>
+    {{ _('Add-on user changes history') }}
```
Comment (clouserw) [inline apps/editors/templates/editors/review.html]: "changes" should be singular.  "Add-on user change history"

File: apps/users/templates/users/email/author_added.ltxt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{% load i18n %}{# L10n: This is an email. Whitespace matters! #}{% blocktrans with addon_url=addon.get_absolute_url addon_name=addon.name author_url=author.user.get_absolute_url author_name=author.user.name %}
+The following author has been added to your addon <a href="{{ addon_url }}">{{ addon_name }}</a>:
+
+<a href="{{ author_url }}">{{ author_name }}</a>
```
Comment (clouserw) [inline apps/users/templates/users/email/author_added.ltxt]: Seems like this should have the role as well?

**Comment by clouserw** [comment]
r+wc, let's get QA to verify on -dev

**Pull request merged**